python/ray/ml/train/integrations/sklearn/sklearn_trainer.py

```diff
@@ -0,0 +1,133 @@
+
+            import ray
+
+            from ray.ml.train.integrations.sklearn import SklearnTrainer
+            from sklearn.ensemble import RandomForestRegressor
+
+            train_dataset = ray.data.from_items(
+                [{"x": x, "y": x + 1} for x in range(32)])
+            trainer = SklearnTrainer(
+                sklearn_estimator=RandomForestRegressor,
+                label_column="y",
+                scaling_config={
+                    "trainer_resources": {"CPU": 4}
+                },
+                datasets={"train": train_dataset}
+            )
+            result = trainer.fit()
+
+    Args:
+        estimator: A scikit-learn compatible estimator to use.
+        datasets: Ray Datasets to use for training and validation. Must include a
+            "train" key denoting the training dataset. If a ``preprocessor``
+            is provided and has not already been fit, it will be fit on the training
+            dataset. All datasets will be transformed by the ``preprocessor`` if
+            one is provided. All non-training datasets will be used as separate
+            validation sets, each reporting separate metrics.
+        label_column: Name of the label column. A column with this name
+            must be present in the training dataset.
+        scoring: Strategy to evaluate the performance of the model on
+            the validation sets and for cross-validation. Same as in
+            ``sklearn.model_selection.cross_validation``.
+            If ``scoring`` represents a single score, one can use:
+
+            - a single string;
+            - a callable that returns a single value.
+
+            If ``scoring`` represents multiple scores, one can use:
+
+            - a list or tuple of unique strings;
+            - a callable returning a dictionary where the keys are the metric
+            names and the values are the metric scores;
+            - a dictionary with metric names as keys and callables a values.
+
+        cv: Determines the cross-validation splitting strategy. If specified,
+            cross-validation will be ran on the train dataset, in addition to
+            computing metrics for validation datasets. Same as in
+            ``sklearn.model_selection.cross_validation``, with the exception of
+            None.
+            Possible inputs for ``cv`` are:
+
+            - None, to skip cross-validation.
+            - int, to specify the number of folds in a ``(Stratified)KFold``,
+            - ``CV splitter``,
+            - An iterable yielding (train, test) splits as arrays of indices.
+
+            For int/None inputs, if the estimator is a classifier and ``y`` is
+            either binary or multiclass, ``StratifiedKFold`` is used. In all
+            other cases, ``KFold`` is used. These splitters are instantiated
+            with ``shuffle=False`` so the splits will be the same across calls.
+        scaling_config: Configuration for how to scale data parallel training.
```

Comment:
This won't be doing data parallel training, is that right?

Reply:
No, it's just for validation. It will all be ran on a single actor.

Reply:
Should we then change the docstring to not mention data parallel training?

Reply:
Yes!